overlord/snapstate/flags.go

```diff
@@ -108,6 +108,11 @@ type Flags struct {
 
 	// Lane is the lane that tasks should join if Transaction is set to "all-snaps".
 	Lane int `json:"lane,omitempty"`
+
+	// Prefer enables all aliases of the given snap in preference to
+	// conflicting aliases of other snaps whose aliases will be disabled
+	// (or removed, for manual ones).
```

Comment:
See other nitpick comment on description

Reply:
Updated, Thank you!